### PR TITLE
fix: Add yellow slanted line back for edX only

### PR DESCRIPTION
**Pull request by attiyaIshaque**
### Description 
https://2u-internal.atlassian.net/browse/VAN-1099
### Add yellow slanted line back for edX only
|BEFORE | AFTER |
|------|--------|
Screen Shot 2022-10-05 at 2 54 15 PM|Screen Shot 2022-10-05 at 2 51 34 PM
Screen Shot 2022-10-05 at 2 54 47 PM|Screen Shot 2022-10-05 at 2 50 47 PM
Screen Shot 2022-10-05 at 2 54 38 PM|Screen Shot 2022-10-05 at 2 51 02 PM

### CODE REVIEW

File: .env.development [hunk 1]
```diff
@@ -17,7 +17,7 @@ MARKETING_SITE_BASE_URL='http://localhost:18000'
 ORDER_HISTORY_URL='http://localhost:1996/orders'
 REFRESH_ACCESS_TOKEN_ENDPOINT='http://localhost:18000/login_refresh'
 SEGMENT_KEY=''
-SITE_NAME='Your Platform Name Here'
+SITE_NAME=''
```
Comment (shafqatfarhan) [inline .env.development:20]: @attiyaIshaque Is this removed intentionally?

File: src/base-component/LargeLayout.jsx [hunk 2]
```diff
@@ -3,6 +3,7 @@ import React from 'react';
 import { getConfig } from '@edx/frontend-platform';
 import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
 import { Hyperlink, Image } from '@edx/paragon';
+import classNames from 'classnames';
```
Comment (shafqatfarhan) [inline src/base-component/LargeLayout.jsx:6]:
```suggestion
import React from 'react';
import classNames from 'classnames';

import { getConfig } from '@edx/frontend-platform';
import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
import { Hyperlink, Image } from '@edx/paragon';
```

File: src/sass/_base_component.scss [hunk 3]
```diff
@@ -188,3 +188,24 @@ $accent-a-light: #c9f2f5;
...
+  border: 4px solid #F0CC00;;
+  transform: rotate(102.02deg);
+}
+
+.small-line {
+  width: 80px;
+  height: 0;
+  border: 4px solid #F0CC00;;
+  transform: rotate(102.02deg);
+}
```
Comment (shafqatfarhan) [inline src/sass/_base_component.scss:211]: EOF line missing

File: src/sass/_base_component.scss [hunk 4]
```diff
@@ -188,3 +188,24 @@ $accent-a-light: #c9f2f5;
   border: 3px solid #F0CC00;
   transform: rotate(102.02deg);
 }
+
+.large-line {
```
Comment (shafqatfarhan) [inline src/sass/_base_component.scss:192]: Can we please rename to: `.large-slanted-line`? Same for medium-line and small-line class names.

File: .env.development (hunk 1 above)
Comment (attiyaIshaque) [reply .env.development]: @shafqatfarhan Remove by mistake.

File: src/base-component/LargeLayout.jsx (hunk 2 above)
Comment (attiyaIshaque) [reply src/base-component/LargeLayout.jsx:6]: Added.

File: src/sass/_base_component.scss (hunk 3 above)
[the remark this answers is not in the document]
Comment (attiyaIshaque) [reply src/sass/_base_component.scss]: It can't be used in our case.

**Review by shafqatfarhan — APPROVED** [review]
👍

**Pull request merged**